plugins/topology/src/common/components/Status.tsx

```diff
@@ -36,13 +40,39 @@ const DASH = '-';
 const Status = ({
   status,
   iconOnly,
+  displayStatusText,
 }: React.PropsWithChildren<StatusProps>): React.ReactElement => {
   const statusProps = {
     title: status ?? '',
```

Comment:
You donot need to pass `displayStatusText` as a prop to `StatusIconAndText`

```suggestion
    title: displayStatusText || status || '',
```